proton/cert_pinning.py

```diff
@@ -0,0 +1,98 @@
+import base64
+import hashlib
+import json
+import os
+
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.connectionpool import HTTPSConnectionPool
+from requests.packages.urllib3.poolmanager import PoolManager
+
+crypto = requests.packages.urllib3.contrib.pyopenssl.OpenSSL.crypto
+
+class InspectedHTTPSConnectionPool(HTTPSConnectionPool):
```

Comment:
I am not sure about this name `InspectedHTTPSConnectionPool`. I.e. Pool of Inspected HTTPS Connections.  Thing is not only the connections are Inspected, but also Verified. However, not all of them but only those for which TLS pins are available. So it's more that this Pool supports TLS Pinning. So maybe `TLSPinningHTTPSConnectionPool`? Or anyone has a better proposal?

Reply:
Indeed, I agree.